otter/util/fp.py

```diff
@@ -122,3 +122,8 @@ def set_in(mapping, keys, new_value):
     else:
         child = mapping.get(keys[0], pmap())
         return mapping.set(keys[0], set_in(child, keys[1:], new_value))
+
+
+def raise_(e):
```

Comment:
Wait... no.  This already exists in Effect and just shouldn't be implemented, I think? http://effect.readthedocs.io/en/latest/apidocs.html?highlight=catch#effect.raise_

That version also respects exception chaining, so better all around :)

Although the same feedback applies to that name :).